src/harness/harness.ts

```diff
@@ -1702,9 +1702,9 @@ module Harness {
                 expected = IO.readFile(refFileName);
             }
 
-            var lineEndingSensitive = opts && opts.LineEndingSensitive;
+            var lineEndingInsensitive = opts && opts.LineEndingSensitive === false; // default is true

```

Comment:
Who sets this flag? i could not find it used anywhere... if that is correct.. then remove this whole section instead.